core/domain/exp_services_test.py

```diff
@@ -3607,6 +3610,92 @@ def test_delete_state_cmd(self) -> None:
         exploration = exp_fetchers.get_exploration_by_id(self.EXP_0_ID)
         self.assertNotIn('new state', exploration.states)
 
+    def test_delete_state_cmd_rejects_obsolete_translation_suggestions(
+        self
+    ) -> None:
+        """Verify deleting a state name rejects corresponding suggestions."""
+        # Add a new state with content.
+        exploration = exp_fetchers.get_exploration_by_id(self.EXP_0_ID)
+        content_id_generator = translation_domain.ContentIdGenerator(
+            exploration.next_content_id_index
+        )
+        content_id = content_id_generator.generate(
+                    translation_domain.ContentType.CONTENT)
```

Comment:
Should only be indented by 4 from the previous line.

Reply:
Ah, thanks. Done.